.travis.yml

```diff
@@ -25,6 +25,7 @@ script:
         ninja;
         echo ==== Running unit tests;
         ulimit -n 1024;
+        export MESON_TESTTHREADS=16;
```

Comment:
Ugh! I dislike seeing hardcoded values like this but I agree that Meson is overly pessimistic given that the unit are not even close to being CPU bound. I can't see anything in the Meson documentation that would allow us to scale this based on the actual number of cores. So go ahead and merge this with the addition of a `TODO` comment about how this should be scaled. It would probably be better is this was moved out of this specific script to a global scope but that isn't essential.